playwright.config.js

```diff
@@ -15,7 +15,7 @@ import { defineConfig, devices } from '@playwright/test';
 export default defineConfig({
   testDir: './tests',
   /* Run tests in files in parallel */
-  fullyParallel: true,
+  fullyParallel: false,   // disables per-file parallelism
```

Comment:
This is necessary to run Link/Unlink tests and 
other tests that need sequential runs.